config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -234,6 +234,7 @@ presubmits:
             - --
             - --build=quick
             - --cluster=
+            - --env=KUBE_FEATURE_GATES=DisableCloudProviders=false,DisableKubeletCloudCredentialProviders=false
```

Comment:
I don't think we should skew the PR vs CI periodic
/hold

Reply:
Can we go ahead and match the equivalent periodic / release blocking until we're ready to turn it off in PR?

Reply:
I took this to mean `ci-kubernetes-e2e-ubuntu-gce-containerd` should have this same feature gate. updated PR